scripts/page_browse.js

```diff
@@ -470,10 +480,12 @@ function pageBrowse(params, replaceUrl) {
             resetButton.click(function () {
                 selectAProject();
             });
-            fixHead.append($("<p>").append(projectData.title, resetButton));
+            const projectRef = new URL(proofIntData.projectFile);
+            projectRef.searchParams.append("id", projectId);
+            fixHead.append($("<p>").append($("<a>", {href: projectRef}).append(projectData.title)), resetButton);
         }
         // get pages
-        $.ajax(makeApiAjaxSettings("v1/projects/" + projectId + "/pages")).done(displayPages)
+        $.ajax(makeApiAjaxSettings(`v1/projects/${projectId}/pages`)).done(displayPages)
```

Comment:
we're ok since no IE support, right?

Reply:
We have officially dropped IE support. 👍